nixos/modules/installer/cd-dvd/installation-cd-graphical-base.nix

```diff
@@ -33,7 +33,11 @@ with lib;
   powerManagement.enable = true;
 
   # Enable sound in graphical iso's.
-  hardware.pulseaudio.enable = true;
+  services.pipewire = {
+    enable = true;
+    pulse.enable = true;
+    alsa.enable = true;
```

Comment:
alsa is free to enable, right?

Reply:
Almost. It's a few kilobytes at most.

Reply:
Should we than also enable this by default in gnome.nix as a follow up?

Reply:
Also don't we need to disable pulseaudio here because gnome enables it?

Reply:
That's a good question. The Plasma modules default to "no opinion" currently, but maybe we should switch everything to Pipewire?

Reply:
People should expect the live ISO to have the same default configuration as the installed system.

Reply:
I am ok with having this in GNOME and KDE enabled as well.